R/pivot_quantiles.R

```diff
@@ -68,10 +68,10 @@ pivot_quantiles_longer <- function(.data, ...) {
 
 #' Pivot a column containing `quantile_pred` wider
 #'
-#' Any selected columns that contain `quantile_pred` will be "widened" with
-#' the "taus" (quantile) serving as names and the values in the data frame.
-#' When pivoting multiple columns, the original column name will be used as
-#' a prefix.
+#' Any selected columns that contain `quantile_pred` will be "widened" with the
+#' "taus" (quantile) serving as column names and the values in the corresponding
+#' column.  When pivoting multiple columns, the original column name will be
+#' used as a prefix.
```

Comment:
I don't think we have "taus" anymore. Also, can we document these two functions
together https://roxygen2.r-lib.org/articles/reuse.html